packages/fxa-payments-server/src/store/index.tsx

```diff
@@ -71,64 +66,43 @@ export const selectors: Selectors = {
 
 export const actions = createActions(
   {
-    fetchProfile: (accessToken: string) =>
-      apiGet(accessToken, `${config.servers.profile.url}/v1/profile`),
-    fetchPlans: (accessToken: string) =>
-      apiGet(
-        accessToken,
-        `${config.servers.auth.url}/v1/oauth/subscriptions/plans`
-      ),
-    fetchSubscriptions: (accessToken: string) =>
-      apiGet(
-        accessToken,
-        `${config.servers.auth.url}/v1/oauth/subscriptions/active`
-      ),
-    fetchToken: (accessToken: string) =>
-      apiPost(accessToken, `${config.servers.oauth.url}/v1/introspect`, {
-        token: accessToken,
+    fetchProfile: (apiClient: any) =>
+      apiClient.get(`${apiClient.config.servers.profile.url}/v1/profile`),
```

Comment:
I'd also say all these paths should probably be refactored into the `apiClient` module too. Maybe export more descriptive functions like `getProfile()` & `getPlans()` to use here and we can manage these URLs in the `apiClient` if/when they change later.

Reply:
And I've realized that this will strip these functions down further to something like `fetchProfile: () => apiClient.getProfile()` - but simplifying the action creators will probably also make weaning away from redux-actions easier?